.github/workflows/build-push-docker-image.yml

```
@@ -269,8 +269,35 @@ jobs:
+        run: |
+          tag=${{ steps.login-ecr.outputs.registry }}/${{ inputs.image_repo }}:${{ inputs.image_tag }}
+          docker buildx imagetools create -t $tag $tag-amd64 $tag-arm64
+
+      - name: Create latest multi-arch manifest for ECR
+        # To be triggered only for release/hotfix PR merges
+        if: ${{ inputs.create_latest_tag == true }}
+        run: |
+          tag=${{ steps.login-ecr.outputs.registry }}/${{ inputs.image_repo }}:${{ inputs.image_tag }}
+          docker buildx imagetools create -t ${{ steps.login-ecr.outputs.registry }}/${{ inputs.image_repo }}:${{ inputs.build_type == 'ut' && 'ut-' || '' }}latest $tag-amd64 $tag-arm64
+
+  create-manifest-dockerhub:
```

actionlint (1.7.10)

key "create-manifest-dockerhub" is duplicated in "jobs" section. previously defined at line:255,col:3. note that this key is case insensitive

(syntax-check)